### PR TITLE
Implement the discovery function for the Azure extension

### PR DESCRIPTION
**Pull request by bugra-derre**
Resolves: #6

### CODE REVIEW

File: chaosazure/__init__.py [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+def __load_exported_activities() -> List[DiscoveredActivities]:
+    """
+    Extract metadata from actions and probes exposed by this extension.
+    """
+    activities = []
+    activities.extend(discover_actions("chaosazure.fabric.actions"))
+    activities.extend(discover_probes("chaosazure.fabric.probes"))
+    activities.extend(discover_actions("chaosazure.machine.actions"))
+    activities.extend(discover_probes("chaosazure.machine.probes"))
+    return activities
```
Comment (Lawouach) [inline chaosazure/__init__.py]: Might want and extra blank line here :)

**Comment by bugra-derre** [comment]
@Lawouach : I am a little bit confused. I am in trouble with the Merge pull request commit and even worse I am not a Git pro. Is the way I am doing PR via forking a good way? Or should I just open a branch on the original project here?

**Comment by russmiles** [comment]
@bugra-derre did you mean to close this issue without merging?

**Comment by Lawouach** [comment]
No worries @bugra-derre. If your changes are the right ones, simply open up the PR again. The only change I'd like to add is an entry to the changelog. You can make a separate commit for this and we will squash everything during the merge into master.

**Comment by bugra-derre** [comment]
@Lawouach I did another PR. You can close this one. Sorry for my "professional" PR :)

**Pull request closed without merging**